components/CPManalysis.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="w-full h-full">
+    <span class="space-x-3 p-3">
+      <Dropdown v-model="selectedVariable" :options="variables" alttext="Choose a variable." />
+      <Dropdown v-model="selectedResolution" :options="resolution" alttext="Select a resolution." />
+      <Dropdown v-model="selectedCategory" :options="categories" alttext="Choose a category" />
+      <Dropdown v-model="selectedDate" :options="dates" alttext="Select a date." />
+    </span>
+    <div
+      class="bg-no-repeat bg-left-top bg-contain w-full h-full"
+      :style="{backgroundImage: `url(${bgImage})` }"
+    />
+  </div>
+</template>
+
+<script>
+export default {
+  data () {
+    return {
+      selectedVariable: 'pr',
+      selectedResolution: '3KM',
+      selectedCategory: 'FREQ',
+      selectedDate: '2005-05-30',
+      variables: {
+        pr: 'Precipitation'
+      },
+      resolution: {
+        '3KM': '3KM',
+        '12KM': '12KM'
+      },
+      categories: {
+        FREQ: 'FREQ',
+        INT: 'INT'
+      },
+      dates: {
```

Comment:
Notice that the title in the images of INT en FREQ are both "intensity"

Reply:
Corrected!